tests/test_validation.py

```diff
@@ -1083,3 +1054,901 @@ def test_login(self, dmge: DataModelGraphExplorer) -> None:
+            )
+            assert len(errors) == 1
+
+            errors, _ = va_obj.cross_validation(
+                val_rule,
+                Series(["A", "B", "C", "D"], index=[0, 1, 2, 3], name="PatientID"),
+            )
+            assert len(errors) == 1
+
+        with patch.object(
+            schematic.models.validate_attribute.ValidateAttribute,
+            "_get_target_manifest_dataframes",
+            return_value={"syn1": cross_val_df1, "syn2": cross_val_df1},
+        ):
+            errors, _ = va_obj.cross_validation(
+                val_rule, Series(["A", "B", "C"], index=[0, 1, 2], name="PatientID")
+            )
+            assert len(errors) == 1
+
+    def test_cross_validation_match_none_set_rules_passing(
+        self, va_obj: ValidateAttribute, cross_val_df1: DataFrame
+    ):
+        """Tests for cross manifest validation for matchNone set rules"""
+        val_rule = "matchNone Patient.PatientID set error"
+
+        with patch.object(
+            schematic.models.validate_attribute.ValidateAttribute,
+            "_get_target_manifest_dataframes",
+            return_value={"syn1": cross_val_df1},
+        ):
+            assert va_obj.cross_validation(val_rule, Series(["A", "B"])) == (
+                [],
+                [],
+            )
+            assert va_obj.cross_validation(val_rule, Series(["A", "B", "C", "D"])) == (
+                [],
+                [],
+            )
+
+    def test_cross_validation_match_none_set_rules_errors(
+        self, va_obj: ValidateAttribute, cross_val_df1: DataFrame
+    ):
+        """Tests for cross manifest validation for matchNone set rules"""
+        val_rule = "matchNone Patient.PatientID set error"
+
+        with patch.object(
+            schematic.models.validate_attribute.ValidateAttribute,
+            "_get_target_manifest_dataframes",
+            return_value={"syn1": cross_val_df1},
+        ):
+            errors, _ = va_obj.cross_validation(
+                val_rule,
+                Series(["A", "B", "C"], index=[0, 1, 2], name="PatientID"),
+            )
+            assert len(errors) == 1
+
+            errors, _ = va_obj.cross_validation(
+                val_rule,
+                Series(["A", "B", "C", "C"], index=[0, 1, 2, 3], name="PatientID"),
+            )
+            assert len(errors) == 1
+
+    def test_cross_validation_value_match_atleast_one_rules_passing(
+        self, va_obj: ValidateAttribute, cross_val_df1: DataFrame
+    ):
+        """Tests for cross manifest validation for matchAtLeastOne value rules"""
+        val_rule = "matchAtLeastOne Patient.PatientID value error"
+
+        with patch.object(
+            schematic.models.validate_attribute.ValidateAttribute,
+            "_get_target_manifest_dataframes",
+            return_value={"syn1": cross_val_df1},
+        ):
+            assert va_obj.cross_validation(val_rule, Series([])) == ([], [])
```

Comment:
since the rule is match at least one, why in this case it will be passing?

Reply:
This is `value` so the values in the manifest being checked must exist in atleast one other manifest. Since there aren't any it automatically passes.

Reply:
I see. Thanks for explaining